style.css

```diff
@@ -445,4 +445,10 @@ a.wp-block-social-link-anchor::after {
 
 .full-excerpt .posts-list-query .wp-block-post-excerpt__excerpt {
   -webkit-line-clamp: 10 !important;
-}
\ No newline at end of file
+}
+
+/* Sets the cookies block above Optimonk sticky footers which have a z-index: 2147483646  */
+
+.cookie-notice {
+z-index: 3000000000;
+}
```

Comment:
This needs to be put in a `.scss` file then compiled into this file. We can't just write directly into the CSS file unfortunately or it will get overwritten.

Reply:
Really? Could we just put !important on it? If not, would you mind doing the ccss file? I'm not sure how.

Reply:
Yeah it needs to be in an SCSS file, it's not about CSS overriding, it's just that this file will literally get replaced by the stuff generated out of the SCSS. 

I can do it, but somebody else will need to review it. 

I'm curious how testing this in other browsers went as well, because I suspect 3bn won't work, as 2147483646 is, to some degree, the highest number as far as computers are concerned. 

Perhaps chrome can go higher, but I doubt all browsers would, and I suspect that's why the number 2147483646 was chosen by Optimonk.